### DISCUSSION TITLE
ZK Native Bridge to ETH Mainnet now over 5 days, tokens not received

### DISCUSSION THREAD

**Discussion by solidarityacademy** [zkSync-Community-Hub/zksync-developers#758] [General] [answered]
### Environment

Mainnet

### L2 block number

#47665458

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Transaction hash 0x35544743e65ffa363eebd641bc28bcf7c3ef5e85177295ecf63bce9e101b45cf to bridge 0.26 ETH from ZK ERA to ETH Mainnet has taken over 5 days and is still not received. Could an expert or the support team please troubleshoot? Thank you kindly. 

Solidarity

**Comment by alhenaai** [comment]
The transaction you mentioned has been verified on the ZKsync Era network, but it appears that the corresponding transaction has not been found on the Ethereum mainnet. This could be due to several reasons, such as delays or issues in the bridging process.

Here are a few steps you can take:
- **Check the Transaction**: You can view the status of your transaction on the ZKsync Era Explorer [here](https://explorer.zksync.io/tx/0x35544743e65ffa363eebd641bc28bcf7c3ef5e85177295ecf63bce9e101b45cf).
- **Contact Support**: Since the transaction is not showing up on Ethereum, it might be helpful to reach out to the support team of the bridge you used for further assistance.

If you used a third-party bridge, please contact their support directly. If it was through ZKsync's official bridge, you might want to follow up with more detailed inquiries or check if there are known issues affecting bridging times.

For further technical discussions or support, you can also visit the ZKsync developers discussions on Github [here](https://github.com/zkSync-Community-Hub/zksync-developers/discussions).

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by solidarityacademy** [comment]
Now it has been 7 days and still nothing. Please help, it's a meaningful amount for me that is stuck.

**Reply by solidarityacademy** [reply, marked as answer]
Ok it was an issue of claiming. I was able to do so on Syncswap.
